pkg/hsskey/hsskey.go

```diff
@@ -165,14 +167,34 @@ func GetAllHss(warnings io.Writer, verboseDangerous bool, eepromPattern string,
 		}
 	}
 
+	// Attempt to get HSS from files.
 	if hssFiles != "" {
-		hssFileArr := strings.Split(hssFiles, ",")
-		hss, err := GetHssFromFile(warnings, verboseDangerous, hssFileArr, 0)
+		fi, err := os.Stat(hssFiles)
 		if err != nil {
-			io.WriteString(warnings, fmt.Sprintf("Error parsing file %s for HSS: %v\n", hss, err))
-		}
-		if len(hss) > 0 {
-			hssList = append(hssList, hss...)
+			io.WriteString(warnings, fmt.Sprintf("Failed to stat HSS files: %v\n", err))
+		} else {
+			hssFileArr := []string{}
+
+			// Check if hssFiles is a directory or a file. Add all regular files in the base directory.
+			switch mode := fi.Mode(); {
```

Comment:
what happens if it is a symlink?

Reply:
Fair point. I've added a single layer of symlink resolving for the initial link passed in. There should not be a need for recursive directory searches or symlinks inside the directory.